kolibri/plugins/learn/assets/src/modules/recommended/handlers.js

```diff
@@ -77,13 +78,39 @@ function _showLibrary(store, query, channels, baseurl) {
 }
 
 export function showLibrary(store, query, deviceId = null) {
-  if (deviceId) {
-    return setCurrentDevice(deviceId).then(device => {
-      const baseurl = device.base_url;
-      return fetchChannels({ baseurl }).then(channels => {
-        return _showLibrary(store, query, channels, baseurl);
-      });
+  /**
+   * ToDo: remove if block.
+   * Currently the channels & contentnode browser apis in studio
+   * are not able to load content using the the studio base url.
+   * Once studio is updated, this function will need to be refactored
+   * to use the else block code only.
+   *
+   * The if block is only meant for UI viualization purposes only
+   * during development
+   */
+  if (deviceId === KolibriStudioId) {
+    RemoteChannelResource.getKolibriStudioStatus().then(({ data }) => {
```

Comment:
Flagging for any other reviewers that until we have a Studio server with the new public content APIs in them, we have to do it this way.